src/program.js

```diff
@@ -88,11 +88,11 @@ export class Program {
     const cmd = argv._[0];
 
     if (cmd === undefined) {
```

Comment:
None of the exceptions below this line will be caught in the `try` block so their stack traces will not be hidden. I suggest moving the `try` block up above this line so that they are all caught.